hifireg/registration/views/user.py

```diff
@@ -1,23 +1,24 @@
 from django.contrib import messages
 from django.contrib.auth import login
 from django.contrib.auth import views as auth_views
-from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
 from django.shortcuts import render, redirect
 from django.urls import reverse_lazy
 from django.views.generic.edit import FormView
+from django.views import View
```

Comment:
nit: I try to keep imports in alphabetical order within the groups. (based on `from` where the import is coming)
```suggestion
from django.views import View
from django.views.generic.edit import FormView
```

Of course, you've probably observed the groups, but here's the summary anyway:
* python module imports (usually none of these, i.e. `import requests`)
* django imports (i.e. `from django.views import View`)
* absolute app imports (i.e. `from registration.forms import AuthenticationForm`)
* relative imports for helper type stuff (`from .mixins import FunctionBasedView`)